[2025-08-02T11:18:50.603+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-02T11:18:50.662+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pokemon_etl_dag.load_data manual__2025-08-02T11:17:49.708991+00:00 [queued]>
[2025-08-02T11:18:50.675+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pokemon_etl_dag.load_data manual__2025-08-02T11:17:49.708991+00:00 [queued]>
[2025-08-02T11:18:50.677+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-08-02T11:18:50.697+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_data> on 2025-08-02 11:17:49.708991+00:00
[2025-08-02T11:18:50.710+0000] {standard_task_runner.py:63} INFO - Started process 236 to run task
[2025-08-02T11:18:50.715+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pokemon_etl_dag', 'load_data', 'manual__2025-08-02T11:17:49.708991+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/pokemon_dags.py', '--cfg-path', '/tmp/tmp7p6ywco4']
[2025-08-02T11:18:50.721+0000] {standard_task_runner.py:91} INFO - Job 60: Subtask load_data
[2025-08-02T11:18:50.762+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-02T11:18:50.887+0000] {task_command.py:426} INFO - Running <TaskInstance: pokemon_etl_dag.load_data manual__2025-08-02T11:17:49.708991+00:00 [running]> on host 5bcae3e3a0de
[2025-08-02T11:18:51.157+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Luis' AIRFLOW_CTX_DAG_ID='pokemon_etl_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-02T11:17:49.708991+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-02T11:17:49.708991+00:00'
[2025-08-02T11:18:51.159+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-02T11:18:51.348+0000] {load.py:40} INFO - Datos de Pokémon leídos exitosamente desde Parquet.
[2025-08-02T11:18:51.349+0000] {load.py:19} INFO - Intentando conectar a la base de datos PostgreSQL...
[2025-08-02T11:18:51.384+0000] {load.py:25} INFO - Conexión exitosa a la base de datos PostgreSQL.
[2025-08-02T11:18:51.563+0000] {load.py:63} INFO - Datos de Pokémon cargados exitosamente en la base de datos.
[2025-08-02T11:18:51.564+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-02T11:18:51.566+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-02T11:18:51.589+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=pokemon_etl_dag, task_id=load_data, execution_date=20250802T111749, start_date=20250802T111850, end_date=20250802T111851
[2025-08-02T11:18:51.653+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-02T11:18:51.688+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-02T11:18:51.692+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
